packages/Tabs/src/components/Panel/Panel.js

```diff
@@ -5,25 +5,35 @@ import * as sc from "./Panel.styles";
 
 const propTypes = {
   children: PropTypes.node,
-  /** Controls if the option is selected or not */
 
+  /** Controls if the option is selected or not */
   isSelected: PropTypes.bool,
+
+  /** Should unmount or not when isSelected is false */
+  shouldUnmount: PropTypes.bool,
 };
 
 const defaultProps = {
   children: null,
   isSelected: false,
+  shouldUnmount: true,
 };
 
 const Panel = props => {
-  const { isSelected, children, ...moreProps } = props;
+  const { isSelected, shouldUnmount, children, ...moreProps } = props;
 
-  if (!isSelected) {
+  if (!isSelected && shouldUnmount) {
     return null;
   }
 
   return (
-    <sc.Panel {...moreProps} role="tabpanel" data-pka-anchor="tabs.panel">
+    <sc.Panel
+      isSelected={isSelected}
```

Comment:
I guess you should add aria-hidden="true" depending on shouldUnmount too? cc @mikrotron

Reply:
There're a `display:none` in styles.js, so aria-hidden will be unnecessary